### DISCUSSION TITLE
Add a `Merged` selector on Pull Requests UI

### DISCUSSION THREAD

**Discussion by glibsm** [community/community#31270] [Pull Requests]
It's often desirable to see only merged pull requests, rather the ones that have been closed (i.e. rejected, or obsolete). There is currently no easy way to do that other than manually editing the `is: closed` to `is: merge` in the search string.

![image](https://user-images.githubusercontent.com/199077/187300372-a669dedb-b679-4046-9b0c-7f735fcbd982.png)

Perhaps adding a third option Open/Closed/Merged would be a good solution, while keeping all the legacy behavior?

**Comment by queenofcorgis** [comment]
Thanks for the feedback :octocat: - I've labelled this so it has visibility among the proper team.
